devstack/settings

```diff
@@ -47,8 +47,7 @@ if [[ $ENABLE_NFP = True ]]; then
     enable_service nfp_orchestrator
     enable_service nfp_proxy
     enable_service nfp_proxy_agent
-    [[ $NFP_DEVSTACK_MODE = base ]] && enable_service nfp_base_configurator
-    [[ $NFP_DEVSTACK_MODE != base ]] && enable_service nfp_config_orchestrator
+    [[ $NFP_DEVSTACK_MODE = base ]] && enable_service nfp_base_configurator || enable_service nfp_config_orchestrator
```

Comment:
If "enable_service nfp_base_configurator" returns false because of failure in starting base configurator, this will start config orchestrator. This should not be the case right?